src/rewriter/rewrite_db_proof_cons.cpp

```diff
@@ -109,6 +109,11 @@ bool RewriteDbProofCons::prove(CDProof* cdp,
     AlwaysAssert(cdp->hasStep(eqi)) << eqi;
     Trace("rpc-debug") << "- finish ensure proof" << std::endl;
   }
+  if (d_trrc.postProve(cdp, a, b, tid, mid))
```

Comment:
You should only run this if success is false.

Reply:
fixed